### ISSUE TITLE
Learn VueJS

### ISSUE THREAD

**Issue by shakram02** [shakram02/PiFloor#33]
Gain knowledge about how to navigate and use the VueJS framework. Knowledge doesn't need to be deep

**Comment by ahmedhammad97** [comment]
After a little research about what's different in using Vue the way we do, I came to the conclusion that we won't be able create `.vue` files for the components [Link1](https://stackoverflow.com/questions/51550095/how-can-i-use-components-in-a-vuejs2-cdn-project) [Link2](https://stackoverflow.com/questions/53277538/can-i-import-single-file-component-using-vue-and-vue-router-cdn) .

However, we can still workaround it by using components and Vue objects inside typical JS files.
Also, we can still use other Vue libraries [(Vue-Router](https://router.vuejs.org/guide/#html), [Vuex](https://vuex.vuejs.org/guide/)..etc.) the same way used the components (I tested them and they work the same way ✌).
This means we will need a bit more effort to organize the modules ourselves.

Let's keep the progress going 🙌
